docs/DafnyRef/Statements.md

```diff
@@ -748,6 +748,8 @@ expression. For example,
   }
 ```
 
+Unlike `match` statements and `if` expressions, `if` statements do not have to be exhaustive: omitting the `else` block is the same as including an empty `else` block.  To ensure that an `if` statement is exhaustive, use the `if-case` statement documented below.
```

Comment:
I don't think it makes sense to say "if-expressions" are exhaustive, since they are exhaustive by syntax. I think you can remove `and if expressions`.

Reply:
Thanks, done!